bindings/c/test/mako/mako.cpp

```diff
@@ -1573,8 +1573,19 @@ int Arguments::validate() {
 		logr.error("--num_databases ({}) must be >= number of clusters({})", num_databases, num_fdb_clusters);
 		return -1;
 	}
-	if (num_threads < num_databases) {
-		logr.error("--threads ({}) must be >= number of databases ({})", num_threads, num_databases);
+	// In sync mode, threads, and in async mode, async workers are assigned to databases. Having more databases than
+	// threads or async workers leads to unused databases.
```

Comment:
Let us not stray from the established term "asynchronous transactions".
`async_xacts` are concurrent asynchronous transactions "worked on" by the "worker threads" running the event loop (i.e. `io_context`).
To me, therefore, the term `async workers` reads more like the threads running the event loop and not the concurrent asynchronous transactions, and adds to confusion.

Reply:
Let's change it then so

Reply:
Okay, will change it. But is there a more abstract word that describes both the thread workers in the sync mode and the asynchronous transactions in the async mode? I'm asking because in #9189 where I introduced new accumulator classes that accumulate statistics on the actual thread and process levels, so I was looking for a word to replace `ThreadStatistics`.

Reply:
I see originally mako referred to processes as workers. Now we have processes, threads, and in async model some kind of actors. And of course now ThreadStatistics are not really thread-level stastistics in the async model.

I am fine with calling processes as processes, threads as threads, and the smallest units generating the workload as workers. In sync mode there will be a worker per thread and in async mode will be a worker per async transaction. @sfc-gh-jshim do you prefer other naming?

Reply:
@sfc-gh-cwende the fact that it is still named `ThreadStatistics` didn't bother me that much, because even in asynchronous mode, it still refers to statistics from each "thread (flow) of execution". If you're collecting statistics local to the OS thread and would like to rename this class to something else to avoid confusion, I'd be okay with something like `WorkflowStatistics`. `Worker` is a misnomer because 1:1 mapping between "worker thread" and "flow of execution" no longer applies in asynchronous mode, where one worker thread can run 100 concurrent workflows.

@sfc-gh-vgasiunas "worker per thread" is a weird way to call it because it is the thread that's running the work. To call the process "worker process", however, is correct because that process orchestrates the benchmark load assigned to it.

Reply:
Workflow is fine with me too. Let's rename worker to workflow